src/artGenerator/package.json

```
@@ -0,0 +1,16 @@
+{
+  "name": "artgenerator",
+  "version": "1.0.0",
+  "description": "Generate art images.",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "generate:images": "npx ts-node-esm src/main.ts 1 1001"
```

こちら、引数で範囲指定できるようにするのはどうでしょうか？
```suggestion
    "generate:images": "npx ts-node-esm src/main.ts ${npm_config_start} ${npm_config_end}"
```
実行時コマンド以下のようになります
`npm run generate:images -start=1 -end=2`